sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationMockTests.cs

```diff
@@ -61,7 +62,7 @@ public async Task Get()
 
             AssertRequestCommon(request);
             Assert.AreEqual(RequestMethod.Get, request.Method);
-            Assert.AreEqual("https://contoso.appconfig.io/kv/test_key", request.UriBuilder.ToString());
+            Assert.AreEqual(AppendPipelineParams("https://contoso.appconfig.io/kv/test_key"), request.UriBuilder.ToString());
```

Comment:
I would just append api-version to all the strings, it doesn't make them that much longer and makes the test less complicated.